src/utils/authenticate.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
```

Comment:
A few changes here:

1. We should create a file `services/http.js` that will be used to initialize axios. For setting this like the baseURL - because if we just do `axios.post('/api/auth/local' =, ....`, the HTTP request will be sent to `<current url>/api/auth/local` (in this case `http://localhost:3000/api/auth/local` - I suspect that's why you had to add a proxy config :) ) instead of `http://localhost:1337`

2. Create a file `services/auth.js`, and move the login function there. `util/` folder is usually used for some smaller things, like creating some validation functions, or e.g. extracting links from a text string... usually simpler functions that don't depend on network request, user input etc.  - so functions that don't affect any global state